src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard-area-chart/dashboard-area-chart.component.html

```diff
@@ -24,7 +24,7 @@
   </div>
 
   <div class="col-9 d-flex flex-column">
-    <div class="chart mt-3">
+    <div [ngClass]="{'chart mt-3': !isMultiCluster, 'mt-3': isMultiCluster}">
```

Comment:
why this change? if it is to fix the pie chart responsiveness?

Reply:
we donot want the chart class to be applied in case of multi-cluster because it restricts the height of the chart